ports/libpmemobj-cpp/portfile.cmake

```
@@ -1,11 +1,15 @@
-include(vcpkg_common_functions)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO pmem/libpmemobj-cpp
-    REF 1.8
-    SHA512 275d5ec3081ccbc76c57205cc366e213386ff56e5c05fe39e8b980c507967e9f32a31e02d074f554e65cb68509a43c39d143f0443816a89da4a334bd6242ba85
+    REF 8ff8c542a87a6ac9cb52c5c04def66d844c290cb

```

```suggestion
    REF 8ff8c542a87a6ac9cb52c5c04def66d844c290cb #v1.10
```
Could you please add the comments here?